internal/alert/manager.go

```diff
@@ -135,13 +136,36 @@ func (am *alertManager) handlePagerDutyPost(alert core.Alert) error {
 			return fmt.Errorf("client %s could not post to pagerduty: %s", pdc.GetName(), resp.Message)
 		}
 
-		am.logger.Debug("Successfully posted to ", zap.Any("resp", resp))
+		am.logger.Debug("Successfully posted to PagerDuty", zap.Any("resp", resp))
 		am.metrics.RecordAlertGenerated(alert, core.PagerDuty, pdc.GetName())
 	}
 
 	return nil
 }
 
+func (am *alertManager) handleSNSPublish(alert core.Alert, policy *core.AlertPolicy) error {
+	event := &client.AlertEventTrigger{
+		Message:  am.interpolator.SlackMessage(alert, policy.Msg),
+		DedupKey: alert.PathID,
+		Severity: alert.Sev,
+	}
+
+	cli := am.cm.GetSNSClient()
```

Comment:
Knit - cli implies command line interface but is being used to represent client - maybe just change var to `client` or `c`

Reply:
[74f2414](https://github.com/base-org/pessimism/pull/200/commits/74f241413764e3b9caf17a604ffe27f2bc28480c)